ujcatapi/config.py

```diff
@@ -36,3 +36,12 @@ def _get_comma_separated_env_variable(name: str) -> List[str]:
 # ====== Feature flags ======
 ENABLE_FOO = _get_boolean_env_variable("ENABLE_FOO")
 ENABLE_BAR = _get_boolean_env_variable("ENABLE_BAR")
+
+ALLOWED_ORIGINS_REGEXPES = [
+    "http://localhost[^\\.]*$",
+    "https://ai\\.gengo\\.test$",
+    "https://.*\\.telusinternational\\.ai$",
+    "https://.*\\.lionbridgeai\\.com$",
+    "https://.*\\.lionbridge\\.com$",
+    "capacitor://localhost$",
```

Comment:
[suggestion] Instead of keeping all domains for "just in case", it is better to only leave what is really needed (YAGNI).
```suggestion
```